config.py

```diff
@@ -0,0 +1,109 @@
+### configuration file for odf_process_all_MK.py
+# TODO: organize these by editable/fixed variables
+
+# Cruise specifics
+# not really needed for processing, move elsewhere
+cruise = dict(
```

Comment:
Put marker as dummy config for customization on future cruises

Reply:
can you clarify what you mean?